src/blob/index/simple.rs

```diff
@@ -10,11 +12,12 @@ pub(crate) struct Simple {
     ioring: Option<Rio>,
 }
 
-#[derive(Debug, Deserialize, Default, Serialize, Clone)]
+#[derive(Debug, Deserialize, Serialize, Clone)]
 pub(crate) struct IndexHeader {
     pub records_count: usize,
     pub record_header_size: usize,
     pub filter_buf_size: usize,
+    pub hash: Vec<u8>,
```

Comment:
Index header has changed, so old index files are no longer valid. This is problem. In the current state we can ignore it, but I suggest to add a version field at the very beggining of the index header. This will allow to distinguish between versions in the future

Reply:
@pyakushin what do you think?